### DISCUSSION TITLE
istio-ingressgateway supported in Linkerd?

### DISCUSSION THREAD

**Discussion by shahriak** [linkerd/linkerd2#6093] [Q&A]
Hello everyone,

Is it possible use istio ingressgateway as the ingress for Linkerd? I am not seeing it as one of the suggested ingresses in the docs https://linkerd.io/2.10/tasks/using-ingress/. If yes, is there any performance implications we should be aware of?

**Comment by wmorgan** [comment]
I don't know if anyone has done it, but I don't know of any reason offhand why it wouldn't work. (With the caveats around Host headers, etc, as discussed in that document you point to.)

What's the motivation for this ingress controller as opposed to something else?

**Reply by shahriak** [reply]
No reason in particular. We have mostly been using istio-ingressgateway for Istio and would like to use the same for Linkerd as well.

**Reply by wmorgan** [reply]
Ok, please let me know if you get it to work. That would be really interesting and we could add it to the docs.
